Dockerfile.full-rp-dev

```diff
@@ -0,0 +1,42 @@
+FROM registry.access.redhat.com/ubi8/toolbox:8.10
+
+# Needed for installing packages
+USER root 
```

Comment:
does this have to be root?

Reply:
I think it was important in the previous tests, but not anymore.